client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -41,18 +48,50 @@ class RequiredPluginsInstallView extends Component {
 			toInstall: [],
 			workingOn: '',
 			stepIndex: 0,
+			numTotalSteps: 0,
+			requestedTransferStatus: false,
 		};
 		this.updateTimer = false;
+		this.transferStatusFetcher = null;
 	}
 
 	componentDidMount = () => {
+		const { signupIsStore } = this.props;
+
 		this.createUpdateTimer();
+
+		if ( signupIsStore ) {
+			this.fetchAutomatedTransferStatus();
+			this.startSetup();
+		}
 	};
 
 	componentWillUnmount = () => {
 		this.destroyUpdateTimer();
 	};
 
+	fetchAutomatedTransferStatus = () => {
+		const { signupIsStore, automatedTransferStatus } = this.props;
+		const { requestedTransferStatus } = this.state;
+
+		if ( signupIsStore && ! requestedTransferStatus ) {
+			this.props.fetchAutomatedTransferStatus( this.props.siteId );
+
+			if ( ! automatedTransferStatus ) {
+				this.transferStatusFetcher = delay( this.fetchAutomatedTransferStatus, 4000 );
+
+				return;
+			}
+
+			this.setState( {
+				engineState: 'DOING_TRANSFER',
+				requestedTransferStatus: true,
+				stepIndex: 1,
+				numTotalSteps: 6,
```

Comment:
Please add a comment explaining the 6 and what sort of changes in the code might require a change to this value.